### ISSUE TITLE
Feature request: Scan robotframework library to use for Keyword Autocomplete feature instead of dictionary

### ISSUE THREAD

**Issue by idxn** [KMK-ONLINE/vscode-robotExtension#1]
First of all, I would like to say thank you for your contribution for this plugin. It is really good and I really love this plugin. I would like to ask for a feature as title.
Let's say we have configuration like this.

["AppiumLibrary", "Selenium2Library", "Process"]

It will scan all the specified library and will be included in keyword suggestions.

**Comment by hainayanda** [comment]
So what you mean it will automatically scan the installed library inside your pc, or you need to manually configure which library would be included?

**Comment by idxn** [comment]
I prefer to the manual one and have a specific configuration for each workspace because on some project we may use only selenium2library so appiumlibrary should not be included.

However, if you implement to automatically scan all installed library and implement a configuration to hide/show somes in suggestion, I'm ok with that as well.

**Comment by hainayanda** [comment]
The extension itself search for library inside your file or included resources. So if your file or file's resources have "library appiumLibrary" words inside the settings, it will search the dictionary for appiumLibrary keywords and show it on suggestion, and youre not gonna see any selenium2library keywords in your files if youre not included it in your settings. Thats why Im a bit confused why do you need more configuration for it.

But if you need the extension to search the library automatically without the dictionary so it more dynamic, and by using a config, it can be done.

**Comment by idxn** [comment]
Yes indeed, I need the extension to search the library automatically without the dictionary.

I'm glad we're on the same page. I'm so sorry to make you're confused. It's totally my bad, I think the extension get the suggestion from dictionary only and do not search for installed library included in the settings.

**Comment by hainayanda** [comment]
Okay. Just wait for the update and dont forget to check the changelog. It will be done soon

**Comment by hainayanda** [comment]
theres new update that will let you to create your own config. the automatic scan part are still on progress, which I still searching how the robot framework library is installed for Linux or windows OS.
